packages/skia-draw/src/hooks/useCanvasDraw.ts

```diff
@@ -8,7 +8,7 @@ type CanvasSetup = {
     initialDrawColor: Color,
     initialStrokeWidth: number,
     ref: ForwardedRef<SkiaDrawHandle>,
-    skiaCanvasRef: RefObject<SkiaView>
+    skiaCanvasRef: RefObject<SkiaDomView>
```

Comment:
What is the consequence of changing the type of this ref to `SkiaDomView` instead of `SkiaView` as before ?